### DISCUSSION TITLE
File size

### DISCUSSION THREAD

**Discussion by 1o1developmentadmin** [nolimits4web/swiper#6531] [General]
I am using following code but swiper-element-bundle.min.js is too heavy. One of my requirement is to use un-minified version but that is 350kb+ in size.

Is this possible to use only required modules within js structure instead of whole bundle?

`  <swiper-container class="mySwiper">
    <swiper-slide>Slide 1</swiper-slide>
    <swiper-slide>Slide 2</swiper-slide>
    <swiper-slide>Slide 3</swiper-slide>
    <swiper-slide>Slide 4</swiper-slide>
    <swiper-slide>Slide 5</swiper-slide>
    <swiper-slide>Slide 6</swiper-slide>
    <swiper-slide>Slide 7</swiper-slide>
    <swiper-slide>Slide 8</swiper-slide>
    <swiper-slide>Slide 9</swiper-slide>
  </swiper-container>
`
  <script src="https://cdn.jsdelivr.net/npm/swiper@9/swiper-element-bundle.min.js"></script>

**Comment by robinsimonklein** [comment]
According to the documentation, you can import only Core of swiper, add your required modules and then initialize it manually : https://swiperjs.com/element#core-version-and-modules
